airflow/dags/ntd_report_validation/a10_submitted_for_ntd.yml

```diff
@@ -0,0 +1,7 @@
+operator: operators.BlackCatApiToGCSOperator
+
+bucket: "gs://calitp-ntd-report-validation"
+api_url: "https://services.blackcattransit.com/api/APIModules/GetNTDReportsByYear/BCG_CA/2023"
```

Comment:
This appears to be the same URL for each DAG task which means the same file is downloaded multiple times. To build on top of [my comment about the downloading and extracting of the data](https://github.com/cal-itp/data-infra/pull/3129/files#r1398015231), it seems like there should be a DAG task to download this file once. Also, a single external table could be used as source data for dbt modeling.

Reply:
changed as suggested